pkgs/development/python-modules/barcodenumber/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-stdnum
+, unittestCheckHook
+,
+}:
+buildPythonPackage rec {
+  pname = "barcodenumber";
+  version = "0.2.1";
+  format = "setuptools";
+
+  # bitbucked repo is no longer public or does no longer exist
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-nW8+m32i42kLR4oC4lrKoBkFYgKHgpZPGAFNJvtLMhc=";
+  };
+
+  patches = [ ./01-fix-test.patch ];
```

Comment:
Is this fixing the only test case there is?

Reply:
iirc, the first of two test cases.